packages/lesswrong/components/posts/ReportPostMenuItem.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { PureComponent } from 'react';
+import { registerComponent, Components } from 'meteor/vulcan:core';
+import MenuItem from '@material-ui/core/MenuItem';
+import PropTypes from 'prop-types';
+import Users from 'meteor/vulcan:users';
+import withUser from '../common/withUser';
+import withDialog from '../common/withDialog'
+import Report from '@material-ui/icons/Report';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import { withStyles } from '@material-ui/core/styles'
+import classNames from 'classnames'
+
+const styles = theme => ({
+  icon: {
+    color: theme.palette.grey[600]
+  },
+  disabled: {
+    color: theme.palette.grey[400],
+    cursor: "default",
+  }
+})
+
+class ReportPostMenuItem extends PureComponent {
```

Comment:
This looks like it was created by copy-pasting `ReportCommentMenuItem`. Can we refactor these to not have so much duplication?